pkgs/applications/networking/cluster/kube-router/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, buildGoPackage, fetchFromGitHub }:
+
+buildGoPackage rec {
+  name = "kube-router-${version}";
+  version = "0.2.3";
+  rev = "v${version}";
+
+  goPackagePath = "github.com/cloudnativelabs/kube-router";
+
+  src = fetchFromGitHub {
+    inherit rev;
+    owner = "cloudnativelabs";
+    repo = "kube-router";
+    sha256 = "1dsr76dq6sycwgh75glrcb4scv52lrrd0aivskhc7mwq30plafcj";
+  };
+
+  buildFlagsArray = ''
+    -ldflags=
+    -X
+    ${goPackagePath}/pkg/cmd.version=${version}
+    -X
+    ${goPackagePath}/pkg/cmd.buildDate=Nix
+  '';
+
+  meta = with stdenv.lib; {
+    homepage = "https://www.kube-router.io/";
+    description = "Kube-router, a turnkey solution for Kubernetes networking.";
```

Comment:
We don't start `description` with the name of the software, for brevity, or end with a period.
> https://nixos.org/nixpkgs/manual/#sec-standard-meta-attributes

Also is there a better description? I actually can't discern what this does from reading it.

Reply:
It's the official description from https://github.com/cloudnativelabs/kube-router.
Is it better to come up with something one self, instead of using the description the authors chose originally?

Reply:
We should simply describe it to the best of our abilities, and If the description provided by the authors is unclear it should be improved.

Usually the author provides a sufficient description so we don't need to create one entirely.